src/actions/users.js

```diff
@@ -1554,6 +1554,45 @@ export function clearUserAccessTokens() {
     };
 }
 
+// Send a getUserAnalytics request for a user (userId) and team id (teamId)
+// Pass a string indicating which datasets to return (name)
+// Can potentially dispatch 4 action types
+//     1. GET_USER_ANALYTICS_REQUEST
+//        - sets requests.users.getUserAnalytics[name].status to 'started'
+//     2. GET_USER_ANALYTICS_FAILURE
+//        - sets requests.users.getUserAnalytics[name].status to 'failure'
+//        - sets requests.users.getUserAnalytics[name].error to error from server
+//     3. GET_USER_ANALYTICS_SUCCESS
+//        - sets requests.users.getUserAnalytics[name].status to 'success'
+//     4. RECEIVED_USER_ANALYTICS
+//        - adds user analytics data returned to entitites.users.analytics
+export function getUserAnalytics(userId, teamId, name) {
+    return async (dispatch, getState) => {
+        // Dispatch GET_USER_ANALYTICS_REQUEST action
+        dispatch({type: UserTypes.GET_USER_ANALYTICS_REQUEST, data: {name}}, getState);
+        let data;
+
+        // Use getUserAnalytics in src/client/client4.js to call api
```

Comment:
This is another case of a comment saying exactly what the next line is with no real additional information.

Reply:
Removed.